contexts/audit/ruby/Makefile

```diff
@@ -0,0 +1,14 @@
+-include ../../prerequisites/tasks.mk
+-include ../../prerequisites/ruby.mk
+
+##>> run bundle audit
+
+.DEFAULT_GOAL := all
+.PHONY: all
```

Comment:
I believe the 'ignore' task was forgotten here.

Reply:
Yes, I forgot it. But now, as said in the previous comment, this target does not exist anymore.